test/__tests__/userlist-cdk.test.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs';
```

Comment:
Can we have a common function utility for all `-cdk` destinations ?

Reply:
this will be taken care post component test ts migration, I just want to migrate with minimal effort first then refactor and improve later

Reply:
Got it 👍🏼